.github/workflows/update-helm-repo.yaml

```diff
@@ -223,3 +223,20 @@ jobs:
         run: |
           cd helm-charts
           "${CR_TOOL_PATH}/cr" index --config "${CR_CONFIGFILE}" --token "${{ env.AUTHTOKEN }}" --index-path "${CR_INDEX_PATH}" --package-path "${CR_PACKAGE_PATH}" --push
+
+      - name: Login to GHCR
+        uses: docker/login-action@v3.0.0
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ env.AUTHTOKEN }}
+
+      - name: Push charts to GHCR
+        run: |
+          shopt -s nullglob
+          for pkg in .cr-release-packages/*.tgz; do
```

Comment:
Let's be in sync with the github release and upload `${{ env.CR_PACKAGE_PATH }}/${{ steps.parse-chart.outputs.packagename }}.tgz` instead of the for loop. See "Make github release" step.

Later we'll have to deal with provenance as well and the workflow starts by verifying that only one chart needs release. I made it that way to make sure the workflow is simple.

Reply:
Sure, I updated this chunk. Please, take another look